thrift/thrift.bzl

```diff
@@ -2,6 +2,8 @@
 
 _thrift_filetype = FileType([".thrift"])
 
+ThriftInfo = provider(fields=["srcs", "transitive_srcs", "external_jars", "transitive_external_jars"])
```

Comment:
nit: format this with each field on a new line
also consider adding a one line comment to each provider field to document what it's for